src/main/java/frc/team2412/robot/commands/climb/ClimbMidToHighCommand.java

```diff
@@ -2,10 +2,12 @@
 
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import frc.team2412.robot.subsystem.ClimbSubsystem;
+import frc.team2412.robot.subsystem.ClimbSubsystem.ClimbConstants.AutoClimbState;
 
 public class ClimbMidToHighCommand extends SequentialCommandGroup {
 
     public ClimbMidToHighCommand(ClimbSubsystem subsystem) {
+        subsystem.setAutoClimbState(AutoClimbState.MID_HIGH);
```

Comment:
Seems like there might be scenarios that could arise with interrupted / paused commands where the programmatic state may not match the robot state since the code changes state at the very beginning of a command.